src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
I think that you are right. If an user want to use a plugin, and the plugin needs to have OSD as a global, then I guess that he needs to include OSD in a way that it ends up exported as window.OpenSeadragon. If you want to import OSD as a module, though, then the plugin needs to be able to handle this situtaion, that is, not to use the object in the window, as it won't be there.